project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,354 @@
+                          # Another option: $938. $670 for EF 19.9 20.9 Cu.Ft. Refrigerator in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 17.4 # 17.4 from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=278&actionId=1044&bcId=4261
+                       # 15 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_extra_refrigerator
+        option: Misc Extra Refrigerator|EF 19.9
+        apply_logic:
+          - or:
+            - Misc Extra Refrigerator|EF 6.9, National Average
+            - Misc Extra Refrigerator|EF 6.9, Average Installed
+            - Misc Extra Refrigerator|EF 15.9, Fed Standard, bottom freezer-reference fridge
+            - Misc Extra Refrigerator|EF 19.8, bottom freezer
+            - Misc Extra Refrigerator|EF 6.7
+            - Misc Extra Refrigerator|EF 10.2
+            - Misc Extra Refrigerator|EF 10.5
+            - Misc Extra Refrigerator|EF 15.9
+            - Misc Extra Refrigerator|EF 17.6
+        costs:
+          - value: 2003.0 # price1: $1699-1799 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in BestBuy
+                          # price2: $8144-8599 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in Home Depot. This price is too high. Therefore, will not refer these prices. 
+                          # 2003.0=1749*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 1749=(1699+1799)/2
+                          # Another option: $938. $670 for EF 19.9 20.9 Cu.Ft. Refrigerator in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 17.4 # 17.4 from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=278&actionId=1044&bcId=4261
+                       # 15 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_clothes_washer
+        option: Clothes Washer|EnergyStar
+        apply_logic:
+          - Clothes Washer|Standard
+        costs:
+          - value: 1174.3 #price: $ 549.9-$1399 for ENERGY STAR top load clothes washer (>2.5 Cu.Ft) in EnergyStar product databaseprice
+                         # 1174.3=974.45*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 974.45=(549.9+1399)/2
+                         # Another option: $903. $630-660 for replacing clothes washer from MEF0.78 to MEF2 in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 14 # from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=293&actionId=1068&bcId=4538
+                     # 17 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_dishwasher 
+        option: Dishwasher|240 Rated kWh # Based on the EnergyStarV7
+        apply_logic:
```

Comment:
again, not all these options are in baseline, only need to include the ones that are in baseline

Reply:
done